lib/peer-info.js

```diff
@@ -46,6 +48,7 @@ module.exports = class PeerInfo extends EventEmitter {
   }
 
   _getPriority () {
+    if (this.ignoreLimits) return TOP_PRIORITY
```

Comment:
this is too aggressive, we'll be dossing the same nodes on the dht, it has to back off still

Reply:
Not related to the timeout, which as per conversation needs to be much higher for explicit peers that have been attempted > 3 times. Just bumps it ahead of pending connections that can't be tried because we're at the limit.